siemens/questa.mk

```diff
@@ -158,9 +177,19 @@ $(DEP_DIR)/%.questa.o: $(SIM_LIB_DONE) | $(DEP_DIR) $(BLOG_DIR)
 	else echo "Unknown filetype: $(word 2,$^)"; echo "$^"; exit 1; fi; fi;
 	@touch $@
 
+AC_CMD = qverify -c -do $(AC_SCRIPT) -od $(AC_OUT_DIR) -modelsimini $(MS_INI)
+AC_MSG := Running Autocheck
+$(AC_DONE): $(MS_INI) $(DEP_DIR)/$(_TOP).questa.o $(precomp_hook) $(AC_DIRECTIVES)
+	@printf "$(autocheck_str)" > $(AC_SCRIPT)
+	@echo "Autocheck dependencies changed";
+	@$(HDL_BUILD_PATH)/siemens/run_siemens.sh '$(AC_MSG)' '$(AC_CMD)' '$(BLOG_DIR)/autocheck.log';
+	@echo -e "$O Starting autocheck simulation $C (see $(BLOG_DIR)/autocheck.log)"
+	@touch $@
+
+AC_TOTAL = $(shell grep "AC Total" $(AC_OUT_DIR)/autocheck_verify.rpt | tr -s ' ' | cut -d ' ' -f 4)
```

Comment:
Rather than defer evaluate this variable and not ever use it at the `Makefile` level, move this into the `ac_pretty.sh` script where it's actually used. That will also reduce the number of parameters to `ac_pretty.sh` which is a nice side benefit.